code/style.css

```diff
@@ -1,4 +1,288 @@
-html {
-  background: #0025ff;
-  color: #fff;
+body {
+  background:white;
+  color: rgb(78, 77, 77);
+  margin: 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 15px;
+  font-weight: normal;
 }
+
+.navigator {
+  background-color: whitesmoke;
+  height: 125px;
+  padding: 15px 0px;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  position: relative;
+  align-items: center;
+}
+
+.fa-sun {
+  color: rgb(78, 77, 77);
+  font-size: 45px;
```

Comment:
I did not know that we can adjust icons with font-size element! This is a good tip to know, thanks!